docs/source/python/api/datatypes.rst

```diff
@@ -93,6 +93,7 @@ functions above.
 
    DataType
    DictionaryType
+   LargeListType
    ListType
```

Comment:
Could you swap `ListType` and `LargeListType` order because `ListType` is more basic type?

```suggestion
   ListType
   LargeListType
```

Reply:
Done.